### PR TITLE
fate 319341; do not copy lincences to target system

### PR DESCRIPTION
**Pull request by schubi2**

### CODE REVIEW

File: src/lib/installation/clients/copy_files_finish.rb [hunk 1]
```diff
@@ -199,79 +199,6 @@ def main
         # FATE #301937, items are defined in the control file
         SystemFilesCopy.SaveInstSysContent
 
-        # Remove old eula.txt
-        # bugzilla #208908
-        @eula_txt = Builtins.sformat(
```
Comment (schubi2) [inline src/lib/installation/clients/copy_files_finish.rb]: This is sooooo old. :-)

File: package/yast2-installation.changes [hunk 2]
```diff
@@ -1,4 +1,12 @@
 -------------------------------------------------------------------
+Mon May  9 15:14:22 CEST 2016 - schubi@suse.de
+
+- Do not copy licences from inst-sys to target system.
```
Comment (imobach) [inline package/yast2-installation.changes]: np: I'm not sure if we use `license` (US) or `licence` (UK)

**Comment by imobach** [comment]
Only a minor comment. Otherwise, LGTM.

File: src/lib/installation/clients/inst_complex_welcome.rb [hunk 3]
```diff
@@ -388,6 +389,28 @@ def text_mode?
       @text_mode = UI.TextMode
     end
 
+    def license_location
```
Comment (imobach) [inline src/lib/installation/clients/inst_complex_welcome.rb]: It would be nice to have some documentation about this method (just a line is enough) :)

**Comment by imobach** [comment]
LGTM

**Pull request merged**